crates/fs/src/fs.rs

```diff
@@ -301,6 +301,8 @@ impl Fs for RealFs {
             .configure(Config::default().with_poll_interval(latency))
             .unwrap();
 
+        //TODO: unblock this
+        #[cfg(not(target_os = "linux"))]
         watcher
             .watch(path, notify::RecursiveMode::Recursive)
             .unwrap();
```

Comment:
This results in an empty user settings which results in an error during startup. Why was it disabled? 
```
user_settings_content: 
[2024-02-04T23:15:58-08:00 ERROR util] crates/settings/src/settings_file.rs:77: EOF while parsing a value at line 1 column 0
```

Reply:
Can be fixed via [kvark/zed@`daa2c84` (#1)](https://github.com/kvark/zed/pull/1/commits/daa2c8443dc4994700d021f43dea29bdeab8b20d)

Reply:
great stuff, thank you!